ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/RedHat.yml

```diff
@@ -102,11 +102,11 @@
     - (ansible_distribution_major_version == "6") or (ansible_distribution_major_version == "7")
   tags: build_tools
 
-- name: Install additional build tools for RHEL 8
+- name: Install additional build tools for RHEL >= 8
   package: "name={{ item }} state=latest"
   with_items: "{{ Additional_Build_Tools_RHEL8 }}"
```

Comment:
Should we rename this variable to `Additional_Build_Tools_RHEL8Plus`?

Reply:
That sounds good.

Reply:
Yep